tests/console_test.go

```diff
@@ -103,6 +110,33 @@ var _ = Describe("Console", func() {
 				}
 			}, 220)
 
+			It("should close console connection when new console connection is opened", func() {
+				vmi := tests.NewRandomVMIWithEphemeralDisk(tests.RegistryDiskFor(tests.RegistryDiskAlpine))
+
+				RunVMIAndWaitForStart(vmi)
+
+				By("opening 1st console connection")
+				expecter, errChan := OpenConsole(vmi)
+				defer func() {
```

Comment:
hm why `defer func() ...` and not just `defer expecter.Close()`

Reply:
very good question ;) fixed